README.md

```diff
@@ -121,3 +121,5 @@ Please read [CONTRIBUTING.md](#CONTRIBUTING.md) before you help us!
 ### [ akshayparmar90 ](https://github.com/akshayparmar90)
 
 ### [ MEME-MAN234 ](https://github.com/MEME-MAN234)
+
+### [ jmakhack ](https://github.com/jmakhack)
```

Comment:
i think you can remove the header formatting so it doesn't come in big fonts.
just the name and link would do. do this for all contributors.

Reply:
sure, changed the formatting for all of them